tests/display.rs

```diff
@@ -172,6 +180,33 @@ mod structs {
                     .to_string(),
                     "struct",
                 );
+                assert_eq!(
+                    format!(
+                        "{:?}",
+                        Struct {
+                            field1: 0,
+                            field2: 3
+                        }
+                    ),
+                    "Struct: 0 3"
+                );
```

Comment:
This one makes sense. This is meant as a showcase of what DebugCustom is currently useful for.